corehq/apps/smsforms/tasks.py

```diff
@@ -24,6 +25,10 @@ def handle_due_survey_action(domain, contact_id, session_id):
         ):
             return
 
+        if toggles.ONE_PHONE_NUMBER_MULTIPLE_CONTACTS.enabled(domain):
```

Comment:
This will cause the task to get re-queued because another contact is using it, right? A comment here would be useful.

Reply:
Yeah it would cause it not exactly to be "requeued" but just to stay on the queue and continue to be retried regularly. Actually now that you mention it, we may need to release the enqueue lock here (or change it to hold for 1 minute instead of an hour) that is set in `handle_survey_actions`. Otherwise I think (possibly, worth looking harder at) that lock would stay in place for an hour and cause this session to get skipped in processing until the lock expired. We maybe want to try it less often than other things in the queue, but an hour is way to much of a delay I think

Reply:
OK this should be addressed by aeaea58